README.md

```diff
@@ -200,6 +201,55 @@ const updatedHomepage = await homepage.update(
 await homepage.delete();
 ```
 
+### .files
+
+The `files` property provides access to the Strapi Media Library through the Upload plugin. It allows you to retrieve files without directly interacting with the REST API.
```

Comment:
Should we make it obvious it gives access to file info rather than actual files?

```suggestion
The `files` property provides access to the Strapi Media Library through the Upload plugin. It allows you to retrieve files information without directly interacting with the REST API.
```

Reply:
The same applies for description below